qlib/model/base.py

```
@@ -60,7 +61,7 @@ def fit(self, dataset: Dataset, reweighter: Reweighter):
         raise NotImplementedError()
 
     @abc.abstractmethod
-    def predict(self, dataset: Dataset, segment: Union[Text, slice] = "test") -> object:
+    def predict(self, dataset: Dataset, segment: Union[Text, slice] = "test") -> str:
```

Pylint (3.3.7)

[warning] Number of parameters was 3 in 'BaseModel.predict' and is now 3 in overriding 'Model.predict' method

(W0221)